model-server/src/main/kotlin/org/modelix/model/server/handlers/ui/RepositoryOverview.kt

```diff
@@ -98,24 +125,33 @@ class RepositoryOverview(private val repoManager: IRepositoriesManager) {
     }
 }
 
-fun FlowOrInteractiveOrPhrasingContent.buildHistoryLink(repositoryId: String, branchName: String) {
+internal fun FlowOrInteractiveOrPhrasingContent.buildHistoryLink(repositoryId: String, branchName: String) {
     a("../history/${repositoryId.encodeURLPathPart()}/${branchName.encodeURLPathPart()}/") {
         +"Show History"
     }
 }
 
-fun FlowOrInteractiveOrPhrasingContent.buildExploreLatestLink(repositoryId: String, branchName: String) {
+internal fun FlowOrInteractiveOrPhrasingContent.buildExploreLatestLink(repositoryId: String, branchName: String) {
     a("../content/repositories/${repositoryId.encodeURLPathPart()}/branches/${branchName.encodeURLPathPart()}/latest/") {
         +"Explore Latest Version"
     }
 }
 
-fun FlowContent.buildDeleteForm(repositoryId: String) {
+internal fun FlowContent.buildDeleteRepositoryForm(repositoryId: String) {
     form {
         postButton {
             name = "delete"
             formAction = "../v2/repositories/${repositoryId.encodeURLPathPart()}/delete"
```

Comment:
I think we have the same problem as with the other confirmation modal here: after deleting, the user might see a blank page? Or not?

Reply:
The browser stays on the same page, but the user needs a manual reload to see the removed repository. Should I also add the auto-refresh here?

Reply:
Not seeing the effect of an action is probably a UX anti-pattern. So yes, a force reload would be good.